x-pack/plugins/global_search_providers/public/providers/get_app_results.ts

```diff
@@ -117,6 +148,8 @@ const flattenDeepLinks = (
         ...deepAppLink,
         // shift current sublink title into array of sub-sublink titles
         subLinkTitles: [deepLink.title, ...deepAppLink.subLinkTitles],
+        // combine current sublink keywords into array of sub-link keywords
+        keywords: deepLink.keywords.concat(deepAppLink.keywords),
```

Comment:
NIT: `[...deepLink.keywords, ...deepAppLink.keywords]` for consistency with how other things are concatenated in the file.